File: 1546/1546.py
n = int(input())
max = 0
sum = 0
score = list((input().split(' ')))
for i in range(n):
    if int(score[i]) > max:
        max = int(score[i])
for i in range(n):
    sum += int(score[i])/max * 100
print(sum/n)

File: 1546/1546.cpp
#include <iostream>
using namespace std;
int main() {
	int n, score[1000];
	int max = 0;
	double sum = 0;
	cin >> n;
	for (int i = 0; i < n; i++) {
		cin >> score[i];
		if (score[i] > max)
			max = score[i];
	}
	for (int i = 0; i < n; i++)
		sum += (double)score[i] / max * 100;
	cout << sum / n;
}